doc/gov/chapters/chapter09.md

```diff
@@ -0,0 +1,121 @@
+
+As Technology evolves and the infrastructure evolves with it, less and less of exceptions will be needed and hence the Generation of VNFs/CNFs is expected to evolve with it until VNFs/CNFs are completely compliant to CNTT as shown in **Figure 4** below. 
+
+<p align="center"><img src="../figures/tech_transition_plan.png" alt="Transition" title="Transitional Plan" width="100%"/></p>
+<p align="center"><b>Figure 4:</b> Transitional Plans and evolution Generations within CNTT</p>
+
+<a name="8.2.2"></a>
+### 8.2.2 VNF Evolution Generations
+
+**Non-Conformant**: Big deviation from CNTT direction.
+
+**Gen1** (Consistent): Going to the right direction but still lots of work to do.
+
+**Gen2** (Compliant): Using current-state technology exceptions and may also have permitted technology version exceptions.
+
+**Gen3** (Conformant): Using current-state technology exceptions but not permitted technology version exceptions.
+
+**Gen4** (Fully Conformant): No exceptions at all.
+
+<a name="8.2.3"></a>
+### 8.2.3 CNF Evolution Generations
+
+**Non-Conformant**: Big deviation from CNTT direction.
+
+**Gen1** (Consistent): Going to the right direction but still lots of work to do.
+
+**Gen2** (Compliant): Using current-state technology exceptions and may also have permitted technology version exceptions.
+
+**Gen3** (Conformant): Using current-state technology exceptions but not permitted technology version exceptions.
+
+**Gen4** (Fully Conformant): No exceptions at all.
+
+<a name="8.2.4"></a>
+### 8.2.4 Polices
+
+- At any present Time Point (TP) which is within the date range of a given generation, VNFs/CNFs taking advantage of an exception **that is allowed** for that generation will PASS OVP qualification but with a clear **WARNING** indicating the functionality/feature will be deprecated in future generations.
+- At any present Time Point (TP) which is within the date range of a given generation, VNFs/CNFs taking advantage of an exception **that is NOT allowed** for that generation will FAIL OVP qualification and get an **ERROR** indicating the functionality is prohibited.
+- It is up to **each operator** to introduce a policy to weather to allow VNFs/CNFs with an outdated Generation to be on-boarded into their network or not.
+- VNF/CNF Generation support by MANO is **out of scope** of CNTT.
+  - VNF/CNF Generations are temporary phases and should be managed solely by receiving operators based on their internal policies.
+
+
+<a name="8.3"></a>
+## 8.3 Adoption Strategy
+- Adoption strategy for operators.
+
+<a name="8.4"></a>
+### 8.4 Roadmap
+
+<a name="8.4.1"></a>
+#### 8.4.1 CNTT Roadmap
+
+<a name="8.4.2"></a>
+#### 8.4.2 VNF Adoption Roadmap
+
+| Generation | Details | Date | Notes |
+|------------|----------------|------------|-------|
+| Gen1 | As per Technical Specification |  |  |
+| Gen2 | As per Technical Specification |  |  |
+| Gen3 | As per Technical Specification |  |  |
+| Gen4 | As per Technical Specification |  |  |
```

Comment:
Having multiple phases of generations for VNFs with the same name (e.g., Gen1, Gen2, etc.), pertaining to the same rls of CNTT, but differentiated by the technology they pertain to, seems very confusing. For example, Gen 1 for SRIOV might correspond to CNTT Rls 3, while Gen 1 for feature x-y-z may pertain to CNTT Rls 4 and so on. To keep it simple, I'd consider aligning to a normalized chronology, referenced by each phase of given technology.